packages/lambdas/src/fitbit-auth.ts

```diff
@@ -61,12 +66,12 @@ async function forwardCallToServer(event: APIGatewayEvent) {
  * @returns boolean
  */
 function verifyRequest(event: APIGatewayEvent, secret: string) {
-  const bodyString = JSON.stringify(event.body);
+  if (event.body) {
```

Comment:
Since we validate the body on the handler, maybe we could pass to this function an object with `body: string`? Then we don't need the validation again here.

Something like:

```ts
function verifyRequest({
  body,
  headers,
  secret
}: { 
  body: string;
  headers: APIGatewayProxyEventHeaders;
  secret: string
}) {
  // body is 'string' here
```

Reply:
Very true!